apps/web/src/components/nav/RootNavMenu.tsx

```diff
@@ -148,7 +156,17 @@ export const RootNavMenu: React.FC = () => {
         ></NavMenuLinkButton>
       </NavMenuSection>
       <FreeTrialSidebarWidget />
-      <RootNavMenuFooter />
+      {isV2Enabled ? (
```

Comment:
@antonjoel82 for this particular condition can we use IS_V2_EXPERIENCE_ENABLED FF? instead of IS_V2_ENABLED?

The rest of the conditions should stay the same using the IS_V2_ENABLED condition

Reply:
Sure. Could you please explain the difference? This is where I think using selectors is important so we centralize logical distinctions with helpful comments

Reply:
Yes, the name is slightly confusing. But:
- is v2 enabled - Should only be enabled for new signups after date X (this hides some existing pages like changes page, layout page, etc....)
- is v2 experience enabled - Will be enabled for all users regarding their signup date